src/Control/Lens/TH.hs

```diff
@@ -86,7 +86,8 @@ import Control.Applicative
 #if !(MIN_VERSION_template_haskell(2,7,0))
 import Control.Monad (ap)
 #endif
-import qualified Control.Monad.Trans as Trans
+import Control.Monad.Trans.Class
+import Control.Monad.Trans.State
```

Comment:
NB: I used lazy `State` here simply because that appeared to be the convention when I arrived here (it already uses lazy `Writer`). I'd have no objections to making it strict, however.